### PR TITLE
Grammar and message tweaks

### PR DESCRIPTION
**Pull request by ghost**
:cl: Xhuis
spellcheck: The names of several objects, such as the holopad and biogenerator, have been lowercased.
spellcheck: Added some descriptions to a few objects that lacked them.
/:cl:

Full list of changes:
* The holopad's name has been lowercased.
* The holopad's activation message now has the `notice` span, and has been slightly reworded. ("flicks to life right before your eyes!" -> "flickers to life before your eyes!")
* The biogenerator's name has been lowercased.
* The Codex Gigas is now referred to as "a Codex Gigas" in examine text instead of just "Codex Gigas."
* The Codex Gigas now has a description. ("A book documenting the nature of devils.")
* Properly capitalized the `title` var of the Codex Gigas and made it play nice with other messages.
* Lowercased the names of the protolathe, destructive analyzer, and circuit imprinter.
* The experimentor's name is now improper.
* The experimentor now has a description. ("A 'replacement' for the destructive analyzer with a slight tendency to catastrophically fail.")
* Using the Charge spell on a spellbook now displays the first message in the correct red color.
* Reworded one of the taunt messages when using Charge on a spellbook. ("CUTE!" -> "CUTE! VERY CUTE!")

If there are any other capitalized objects around the station that shouldn't be, let me know and I'll fix them up.

### CODE REVIEW

File: code/modules/library/lib_codex_gigas.dm [hunk 1]
```diff
@@ -29,7 +30,7 @@
 	if(ishuman(user))
 		var/mob/living/carbon/human/U = user
 		if(U.check_acedia())
-			to_chat(user, "<span class='notice'>None of this matters, why are you reading this?  You put the [title] down.</span>")
+			to_chat(user, "<span class='notice'>None of this matters, why are you reading this?  You put [title] down.</span>")
```
Comment (Cyberboss) [inline code/modules/library/lib_codex_gigas.dm]: Extra space after ?

**Review by Cyberboss — CHANGES_REQUESTED** [review]

File: code/modules/library/lib_codex_gigas.dm (hunk 1 above)
Comment (Xhuis) [inline code/modules/library/lib_codex_gigas.dm]: Oh, missed that!

**Review by Cyberboss — APPROVED** [review]

**Comment by lordpidey** [comment]
I'm not certain we should make Codex Gigas be improper, isn't it kind of like "the bible"?

**Review by lordpidey — CHANGES_REQUESTED** [review]
Might as well just remove this line entirely.

**Comment by Xhuis** [comment]
I tried to make it "the Codex Gigas" but I couldn't actually figure out how. 🙁

File: code/modules/library/lib_codex_gigas.dm [hunk 2]
```diff
@@ -91,7 +92,7 @@
 	else if(GLOB.devil_suffix.Find(action))
 		currentSection = SUFFIX
 	else
-		to_chat(world, "Codex gigas failure [action]")
+		to_chat(world, "Codex Gigas failure [action]")
```
Comment (lordpidey) [inline code/modules/library/lib_codex_gigas.dm]: Just remove this line and the else above it, There's a potential for anyone to spam the server with a codex gigas and a html request forger.

**Pull request merged**